packages/synapse-react-client/src/lib/containers/dataaccess/SubmissionPage.tsx

```diff
@@ -62,15 +62,15 @@ function DataAccessSubmissionFileHandleLink(props: {
 }
 
 function ApproveConfirmationModal(
-  props: Pick<WarningModalProps, 'show' | 'onConfirm' | 'onCancel'>,
+  props: Pick<WarningDialogProps, 'open' | 'onConfirm' | 'onCancel'>,
 ) {
   return (
-    <WarningModal
-      show={props.show}
+    <WarningDialog
+      open={props.open}
       title="Approve Request?"
-      modalBody={
+      content={
         <>
-          <Typography variant="body1">
+          <Typography variant="body1" sx={{ marginBottom: '10px' }}>
```

Comment:
Add margin between paragraphs since margin-bottom styling is no longer applied by to `p` by `bootstrap-4-backport`